extensions/context-builder/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "context-builder",
+  "displayName": "%contextBuilder.displayName%",
+  "description": "%contextBuilder.description%",
+  "version": "0.1.0",
+  "publisher": "Microsoft",
+  "preview": true,
+  "engines": {
+    "vscode": "*",
+    "azdata": ">=1.45.0"
+  },
+  "license": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/main/LICENSE.txt",
+  "icon": "images/extension.png",
+  "aiKey": "29a207bb14f84905966a8f22524cb730-25407f35-11b6-4d4e-8114-ab9e843cb52f-7380",
+  "activationEvents": [
+    "*"
```

Comment:
Would suggest making a tracking work item to replace this, we should avoid * activations when possible since they affect startup time

Reply:
Sounds good. I'll create a work item to track this.

Reply:
Issue tracking the replacement of "*" activations: https://github.com/microsoft/azuredatastudio/issues/24096